From c844ec39ce69c6e49d603665cabef8bc080be79f Mon Sep 17 00:00:00 2001
From: Katsuya Matsubara <matsu@igel.co.jp>
Date: Thu, 31 May 2012 15:48:52 +0900
Subject: [PATCH 46/73] rcarvin: fix macro definition of the HPS bit for the DMR2 register

The HPS bit in the DMR2 register the must be set if hsync polarity
would be HIGH.
---
 drivers/media/video/rcarvin.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/video/rcarvin.c b/drivers/media/video/rcarvin.c
index fa7d189..6ca8431 100644
--- a/drivers/media/video/rcarvin.c
+++ b/drivers/media/video/rcarvin.c
@@ -353,8 +353,8 @@ static void free_buffer(struct videobuf_queue *vq,
 #define VIN_VNDMR2_VPS_ACTIVE_LOW  (0x00000000)
 #define VIN_VNDMR2_VPS_ACTIVE_HIGH (VIN_VNDMR2_VPS)
 #define VIN_VNDMR2_HPS             (0x20000000)  /* Hsync Polarity Select */
-#define VIN_VNDMR2_HPS_ACTIVE_LOW  (VIN_VNDMR2_HPS)
-#define VIN_VNDMR2_HPS_ACTIVE_HIGH (0x00000000)
+#define VIN_VNDMR2_HPS_ACTIVE_LOW  (0x00000000)
+#define VIN_VNDMR2_HPS_ACTIVE_HIGH (VIN_VNDMR2_HPS)
 #define VIN_VNDMR2_CES             (0x10000000)
 		/* Clock Enable polarity Select */
 #define VIN_VNDMR2_FTEV            (0x00020000)
-- 
1.7.0.4

